chromium_src/third_party/blink/public/mojom/webpreferences/web_preferences.mojom

```diff
@@ -8,4 +8,5 @@ module blink.mojom;
 [BraveExtend]
 struct WebPreferences {
   bool force_cosmetic_filtering;
+  bool force_to_hide_media_src_api;
```

Comment:
can we add comments here for both members please?

Reply:
Yep. done.